java/src/jmri/jmrit/logix/configurexml/WarrantManagerXml.java

```diff
@@ -404,8 +487,12 @@ private static ThrottleSetting loadThrottleCommand(Element elem) {
 
         Attribute attr = elem.getAttribute("command");
         String command = null;
-        if (attr != null)
+        if (attr != null) {
             command = attr.getValue();
+        } else {
+            log.error("Command type is null for throttleSetting {} in warrant {}", command, w.getDisplayName());
```

Comment:
Spotbugs complains about this line. The parameter `command` is known to be `null` at this point. So the `@SuppressFBWarnings` needs to be removed.